src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/CertificateAccess/Model/DigitalCovidCertificateWithHeader.swift

```diff
@@ -0,0 +1,82 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+public struct DigitalCovidCertificateWithHeader: Codable, Equatable {
+    let header: CBORWebTokenHeader
+    let certificate: DigitalCovidCertificate
+
+    static func fake(
```

Comment:
Maybe wrap this into `#ifdef ` so it does not get used in prod somewhere?

Reply:
Done. Also added them to other mock, fake stuff in the lib.